### PR TITLE
Update to 0.2.1

### PR DESCRIPTION
**Pull request by yinweisu**
Issue #, if available:

Description of changes:
* Noticed that `versions.rst` is not being added to the new website. Will have a follow-up PR so that it's easier to be cherry picked into stable branch

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/versions.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@ Available Documentation for AutoGluon-Cloud
 
 Web-based documentation is available for versions listed below:
 
-- `AutoGluon 0.1.1 (dev) documentation <https://auto.gluon.ai/cloud/dev/index.html>`_
-- `AutoGluon 0.1.0 (stable) documentation <https://auto.gluon.ai/cloud/stable/index.html>`_
+- `AutoGluon 0.2.1 (dev) documentation <https://auto.gluon.ai/cloud/dev/index.html>`_
+- `AutoGluon 0.2.0 (stable) documentation <https://auto.gluon.ai/cloud/stable/index.html>`_
+- `AutoGluon 0.2.0 documentation <https://auto.gluon.ai/cloud/0.2.0/index.html>`_
```
Comment (gidler) [inline docs/versions.rst:8]: Do you want `0.2.0` to have 2 lines here? Its not a big deal but it is different than how we do it in AutoGluon

File: README.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Currently, AutoGluon-Cloud supports [AWS SageMaker](https://aws.amazon.com/sagem
 # First install package from terminal:
 # pip install -U pip
 # pip install -U setuptools wheel
-# pip install --pre autogluon.cloud  # You don't need to install autogluon itself locally
+# pip install autogluon.cloud==0.2.0  # You don't need to install autogluon itself locally
```
Comment (gidler) [inline README.md:20]: Why specify the specific version?

Comment (yinweisu) [reply README.md:20]: Trying to follow what autogluon has: https://github.com/autogluon/autogluon/blob/master/README.md

File: docs/versions.rst (hunk 1 above)
Comment (yinweisu) [reply docs/versions.rst:8]: Huh, I remember I was following AutoGluon. NVM, I'll update it

File: README.md (hunk 2 above)
Comment (gidler) [reply README.md:20]: AutoGluon has `# pip install autogluon  # autogluon==0.7.0` which is also a bit weird

**Review by gidler — APPROVED** [review]

**Pull request merged**